crates/std_detect/src/detect/cache.rs

```diff
@@ -78,7 +78,8 @@ struct Cache(AtomicU64);
 impl Cache {
     /// Creates an uninitialized cache.
     const fn uninitialized() -> Self {
-        Cache(AtomicU64::new(u64::max_value()))
+        const X: AtomicU64 = AtomicU64::new(u64::max_value());
+        Self(X)
```

Comment:
@Centril @oli-obk without this change `std_detect` fails to compile, which is weird. The error is "can only call other min_const_fn within a min_const_fn" but the crate has `feature(const_fn)` enabled =/

Reply:
(repeating from Discord: I wasn't able to reproduce either... I'll see if I have time to test this locally; for the time being, this hack is fine, and `AtomicU64` is stable in 1.34 so hopefully that makes the hack go away... However, when using the hack in the future, please cc myself and @oli-obk...)